File: src/main/java/com/hongwei/model/arcgis/covid19/BeanByAuState.java
package com.hongwei.model.arcgis.covid19;

public class BeanByAuState {
    public String objectIdFieldName;
    public UniqueIdField uniqueIdField;
    public String globalIdFieldName;
    public Fields[] fields;
    public Features[] features;
}

class UniqueIdField {
    public String name;
    public boolean isSystemMaintained;
}

class Fields {
    public String name;
    public String type;
    public String alias;
    public String sqlType;
    public long length = 0;
    public String domain;
    public String defaultValue;
    public String description = "";
}

File: src/main/java/com/hongwei/model/arcgis/covid19/BeanByAuState.kt
package com.hongwei.model.arcgis.covid19

data class BeanByAuState(
        var objectIdFieldName: String,
        var uniqueIdField: UniqueIdField,
        var globalIdFieldName: String,
        var fields: Array<Fields>,
        var features: Array<Features>
)

data class UniqueIdField(
        var name: String,
        var isSystemMaintained: Boolean = false
)

data class Fields(
        var name: String?,
        var type: String?,
        var alias: String?,
        var sqlType: String?,
        var length: Long = 0,
        var domain: String? = null,
        var defaultValue: String? = null,
        var description: String = ""
)